### PR TITLE
mtd_config_fs: support align size test

### PR DESCRIPTION
**Pull request by zhaoxingyu12**
## Summary
The previous test case simulation writing did not consider byte alignment

## Testing
Pass CI.
Local build with the following config, pass.
CONFIG_MTD_CONFIG=y
CONFIG_MTD_BYTE_WRITE=y
CONFIG_MTD_CONFIG_FAIL_SAFE=y
CONFIG_TESTING_MTD_CONFIG_FAIL_SAFE=y
CONFIG_MTD_WRITE_ALIGN_SIZE=8

### CODE REVIEW

File: testing/mtd_config_fs/mtd_config_fs_test_main.c [hunk 1]
```diff
@@ -513,6 +523,18 @@ static void test_nvs_corrupt_expire(struct mtdnvs_ctx_s *ctx)
       goto test_fail;
     }
 
+  padding_size = NVS_ALIGN_UP(sizeof(TEST_KEY1) + sizeof(TEST_DATA1))
+    - sizeof(TEST_KEY1) - sizeof(TEST_DATA1);
```
Comment (acassis) [inline testing/mtd_config_fs/mtd_config_fs_test_main.c:527]:
```suggestion
                - sizeof(TEST_KEY1) - sizeof(TEST_DATA1);

**Comment by acassis** [comment]
@zhaoxingyu12 Please include comments in the commit log, explaining the previous issue.

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by jerpelea — APPROVED** [review]
please add commit message

**Comment by zhaoxingyu12** [comment]
> please add commit message

add commit message done

**Review by acassis — APPROVED** [review]

**Comment by acassis** [comment]
@zhaoxingyu12 please consider submitting a basic documentation about this testing tool

**Comment by zhaoxingyu12** [comment]
> @zhaoxingyu12 please consider submitting a basic documentation about this testing tool

ok, where should I submit it to?

**Pull request merged**